src/utils/store.js

```diff
@@ -44,13 +44,13 @@ function get(key) {
     if (!isLocalStorageSupported) return
     try {
       var value = window.localStorage[`${NAMESPACE}.${key}`]
+
+      if (value) {
+        return JSON.parse(value)
+      }
     } catch (e) {
       return
     }
-
-    if (value) {
-      return JSON.parse(value)
-    }
```

Comment:
Catch as much as possible?
```js
function get(key) {
  try {
    if (getter) {
      return getter(key)
    }

    if (!isLocalStorageSupported) return
    
    var value = window.localStorage[`${NAMESPACE}.${key}`]
    if (value) {
      return _JSON.parse(value)
    }
  } catch (e) {
    return
  }
}
```

Reply:
This seems like a good idea given that on some environments (e.g. iOS Safari in "no cookies" mode) localStorage throws an error if you try to access it.